Core/include/Acts/Propagator/MultiStepperAborters.hpp

```diff
@@ -40,6 +45,7 @@ struct MultiStepperSurfaceReached {
   template <typename propagator_state_t, typename stepper_t>
   bool operator()(propagator_state_t& state, const stepper_t& stepper,
                   const Surface& targetSurface) const {
+    const auto& logger = state.options.logger;
     bool reached = true;
     const auto oldCurrentSurface = state.navigation.currentSurface;
```

Comment:
Probably rename to `previousSurface` ?

Reply:
hmm not sure, since this could still be the current surface, if it turns out that no component is on a surface... I would leave the naming for now...